install/terminal/select-dev-language.sh

```diff
@@ -15,16 +25,22 @@ if [[ -n "$languages" ]]; then
 			;;
 		Node.js)
 			mise use --global node@lts
+
+			enable_lazyvim_extras "lazyvim.plugins.extras.lang.typescript"
 			;;
 		Go)
 			mise use --global go@latest
+
+			enable_lazyvim_extras "lazyvim.plugins.extras.lang.go"
 			;;
 		PHP)
 			sudo add-apt-repository -y ppa:ondrej/php
 			sudo apt -y install php8.3 php8.3-{curl,apcu,intl,mbstring,opcache,pgsql,mysql,sqlite3,redis,xml,zip}
 			php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
 			php composer-setup.php --quiet && sudo mv composer.phar /usr/local/bin/composer
 			rm composer-setup.php
+
+			enable_lazyvim_extras "lazyvim.plugins.extras.lang.php" "lazyvim.plugins.extras.lang.typescript"
```

Comment:
not sure the typescript should be included here ? :thinking:

Reply:
I added it because I usually write JavaScript too in my PHP apps

Reply:
but then the node js option would take care of this, right ?

Reply:
It might, but these days I'm not using Node (using Importmaps)

Reply:
I've removed the TS extras plugin from the PHP installation (I can always enable it myself)